<?xml version="1.0"?>
<launch>

    <arg name="robot" />
    <arg name="rc_path" />

    <include file="$(find cob_bringup_sim)/launch/robot.launch">
        <arg name="gui" value="false"/>
    </include>
    <include file="$(find cob_grasping)/launch/move_group.launch" />

    <test name="grasping_app" pkg="cob_grasping" type="grasping_app.py" output="screen" required="true">
        <param name="scene_config_file" value="$(find cob_grasping)/config/scene_config.yaml" />
        <param name="switch_arm" value="False" />
        <param name="wait_for_user" value="False" />
        <param name="joint_trajectory_speed" value="0.3" />
        <param name="max_error" value="50" />
        <param name="lift_height" value="0.02" />
        <param name="approach_distance" value="0.14" />
        <param name="manipulation_repeats" value="1" />
        <param name="load_obstacles" value="none" />
    </test>

    <node name="atf_recorder" pkg="atf_recorder" type="recorder.py" output="screen">
        <param name="robot_config_file" value="$(arg rc_path)$(arg robot)/robot_config.yaml" />
    </node>

</launch>